superset/dashboards/commands/importers/v1/__init__.py

```diff
@@ -96,7 +100,7 @@ def _import(
                 dataset = import_dataset(session, config, overwrite=False)
                 dataset_info[str(dataset.uuid)] = {
                     "datasource_id": dataset.id,
-                    "datasource_type": "view" if dataset.is_sqllab_view else "table",
+                    "datasource_type": dataset.datasource_type,
```

Comment:
This also caused an import error, since virtual datasets are in fact regular tables (they just have the `sql` property set).